pep-0458.txt

```diff
@@ -457,6 +457,17 @@ pip for the first time.
 
 __ https://docs.google.com/spreadsheets/d/11_XkeHrf4GdhMYVqpYWsug6JNz5ZK6HvvmDZX0__K2I/edit?usp=sharing
 
+This number of bins SHOULD double when the metadata overhead for returning
+users exceeds 50% (relative to the average size of downloaded packages).
+Presently, this SHOULD happen when the number of targets (simple indices
+and packages) increase at least 4x from over 2M to nearly 9M, at which
+point the metadata overhead for returning and new users would be around
+49-54% and 185% respectively, assuming that the number of bins stay fixed.
+If the number of bins is increased, then the cost for all users would
+effectively be the cost for new users. If the cost for new users should
```

Comment:
I don't understand why that's the case.  Could you explain?

Reply:
Because the cost for new users is dominated by the `bins` metadata. Will make a note.